src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -150,23 +179,28 @@ public boolean link() throws IOException, InterruptedException {
         Files.createDirectories(dalvikBinPath);
         Files.createDirectories(dalvikLibPath);
         Files.createDirectories(dalvikLibArm64Path);
-        Path androidManifestPath = dalvikPath.resolve("AndroidManifest.xml");
+        Path androidManifestDalvikPath = dalvikPath.resolve(Constants.MANIFEST_FILE);
```

Comment:
why is this "Dalvik" ?

Reply:
We use an internal folder `gvm/dalvik` where the files to create the apk are added. Maybe we should rename this to something different?

Reply:
yes, this has nothing to do with dalvik (the only dalvik-relevant part we have is where the Activity and related ancient Java files are)

Reply:
OK, filed #359 for that. Any suggestion? `apk`?

Reply:
in this part of the code, since we're referring to the Android Manifest path, I think androidManifestPath is the best name.

Reply:
Ok, changed